bin/epaa.py

```diff
@@ -694,38 +688,18 @@ def get_protein_ids_for_transcripts(idtype, transcripts, ensembl_url, reference)
 def get_matrix_max_score(allele, length):
     allele_model = "%s_%i" % (allele, length)
     try:
-        pssm = getattr(__import__("Fred2.Data.pssms.syfpeithi.mat."+allele_model, fromlist=[allele_model]), allele_model)
-        return sum([max(scrs.values()) for pos, scrs in pssm.iteritems()])
+        pssm = getattr(__import__("epytope.Data.pssms.syfpeithi.mat."+allele_model, fromlist=[allele_model]), allele_model)
+        return sum([max(scrs.values()) for pos, scrs in pssm.items()])
     except:
         return np.nan
 
 
-def create_score_values(j, method):
-    if not pd.isnull(j):
-        if 'syf' in method:
-            return round(j, 4)
-        elif any(m in method for m in ['mhcnuggets','mhcflurry']):
-            # convert IC50 -> affinity score in [0,1]
-            min_ic50 = 1.0
-            return round(1.0 - math.log(max(j,min_ic50), 50000), 4)
-        else:
-            # currently not applied (for methods such as netMHC)
-            return round(j, 4)
```

Comment:
might make sense to keep the `round` for the scores column to get the same results as before.